service/worker/scanner/executions/scavenger.go

```diff
@@ -44,9 +45,8 @@ import (
 )
 
 const (
-	executorPoolSize         = 4
 	executorPollInterval     = time.Minute
-	executorMaxDeferredTasks = 10000
```

Comment:
So we are increasing the # of workers and task buffer size? They are limiting the speed of scavenger?

Reply:
I think it the pool size is the bottleneck. The max deferred task will allow it to scan cluster with large shard number.